object/object.go

```diff
@@ -278,8 +278,8 @@ func (o *Object) PayloadHomomorphicHash() (checksum.Checksum, bool) {
 	v2 := (*object.Object)(o)
 
 	if hash := v2.GetHeader().GetHomomorphicHash(); hash != nil {
-		v.ReadFromV2(*hash) // FIXME(@cthulhu-rider): #226 handle error
-		return v, true
+		err := v.ReadFromV2(*hash)
+		return v, (err == nil)
```

Comment:
I think we may omit `()` in return statements here and in another places below.

Reply:
I'd say it's easier to read this way, two entities returned are clearly seen.